website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -25,6 +25,17 @@ called this endpoint or used this command using tokens with just the `read-job`
 capability or the `read` policy must update their tokens to use the
 `submit-job` capability or the `write` policy.
 
+## Nomad 1.6.0 Enterprise
+
```

Comment:
```suggestion
```

I don't think we add a separate header for enterprise 🤔

Reply:
"Enterprise" shows up rarely in this document in general.  There is precedence with [Nomad 1.0.11 and 1.1.5 Enterprise](https://developer.hashicorp.com/nomad/docs/upgrade/upgrade-specific#nomad-1-0-11-and-1-1-5-enterprise) and I figured this is a pretty significant change for specifically enterprise operators to consider.  In [Nomad 1.1.0](https://developer.hashicorp.com/nomad/docs/upgrade/upgrade-specific#nomad-1-1-0) though, it's a sub-header, so I could be happy with it being the top sub-header like that.

Want me to go ahead and move it under "Nomad 1.6.0" ?

Reply:
Oh yeah, I remember now part of my motivation was to have a more specific `#link-id`, like right now this

 * https://developer.hashicorp.com/nomad/docs/upgrade/upgrade-specific#enterprise-licenses

links to `Nomad 1.1.0 -> Enterprise licenses`, and if I did the same under Nomad 1.6.0 then I'd be changing the destination of that same link.  I suppose this matters less with versioned docs, but `#nomad-1-6-0-enterprise` felt more stable 🤷

Reply:
Oh interesting, the anchor ID is autogenerated, so `Nomad 1.1.0 -> Enterprise licenses` is now `#enterprise-licenses-1`. I don't know how I feel about that 🤔 

Maybe we need a different subtitle from `Enterprise Licenses`, maybe something like `Enterprise Licenses Validation`?

Reply:
Good thinkin, I think I might like

> Enterprise License Validation with BuildDate

which a) stands out nicely, and b) its link ID will probably stay unique

Reply:
How's this look?

https://nomad-2akx4ewo8-hashicorp.vercel.app/nomad/docs/upgrade/upgrade-specific#enterprise-license-validation-with-builddate